libs/service-portal/core/src/lib/service-portal-core.ts

```diff
@@ -4,11 +4,19 @@ import { User } from 'oidc-client'
 import { ServicePortalPath } from './navigation/paths'
 import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
 
+export interface ReactIntlMessage {
```

Comment:
I am wondering if this should be part of the localization package instead?

Reply:
Yeah, that would make sense.

@magnearun should I move it to the localization lib?

Reply:
I think we don't need this interface.

The formatMessage method looks like this formatMessage(descriptor: MessageDescriptor, values: <Record<K, V>>) MessageDescriptor is exported by 'react-intl' as 
MessageDescriptor {
    id?: string | number;
    description?: string | object;
    defaultMessage?: string | MessageFormatElement[];
}

name: ReactIntlMessage | string 

should rather be

name: MessageDescriptor | string